pyproject.toml

```diff
@@ -11,9 +11,7 @@ legacy_tox_ini = """
 
 [testenv:tests]
 changedir = tests
-deps =
-    coverage[toml]
-    parameterized
+deps = -rtests/requirements.txt
```

Comment:
Why would we need a separate file here? You could add tqdm directly in pyproject.toml

Reply:
I added this to be consistent with `examples` and to make it easier to find the testing dependencies.